backend/db/data/books/1.json

```diff
@@ -10,71 +10,71 @@
     },
     {
       "title": "Quidditch",
-      "order": "11"
+      "summary": "Harry joins the Gryffindor Quidditch team as the youngest Seeker in a century. During his first Quidditch match against Slytherin, Harry experiences a series of ups and downs, including Professor Snape's alleged attempt to jinx his broomstick, making it go haywire. Harry's determination prevails and he manages to catch the Golden Snitch, securing the victory for Gryffindor. After the match, Harry, Ron, and Hermione pay a visit to Hagrid to discuss Snape's role in the broomstick incident. Hagrid accidentally reveals information that a three-headed dog is actually his and protects a secret of Dumbledore and Nicholas Flamel."
     },
     {
       "title": "The Mirror of Erised",
-      "order": "12"
+      "summary": "Harry discovers a mysterious mirror in an abandoned classroom at Hogwarts. When he looks into the mirror, he sees himself with his deceased parents, James and Lily Potter, who he never had a chance to know. The mirror shows the deepest desires of a person's heart, and Harry becomes obsessed with returning to it, leading to his late-night visits to the mirror room. Dumbledore eventually intervenes, explaining the mirror's powers and the danger of becoming lost in its illusions."
     },
     {
       "title": "Nicolas Flamel",
-      "order": "13"
+      "summary": "Harry and his friends learn about Nicolas Flamel, a renowned alchemist who is believed to have created the Philosopher's Stone, an object with incredible powers. They discover that Flamel's work with the stone has contributed to his extraordinary longevity, and it's also rumored to produce the Elixir of Life, a potion that grants immortality. Harry accidentally overhears a mysterious conversation between Professor Snape and Professor Quirrell. Snape expresses his distrust of Quirrell's loyalty and his urgent need to know how to access the Philosopher's Stone, which is stored at Hogwarts."
     },
     {
       "title": "Norbert the Norwegian Ridgeback",
-      "order": "14"
+      "summary": "Hagrid wins a dragon egg in a card game and becomes obsessed with hatching it. As the egg is about to hatch, Hagrid asks Harry and Hermione to help him smuggle the dragon, named Norbert, out of the castle to Charlie Weasley, who works with dragons in Romania. The trio's late-night excursion to the tower where Norbert is kept leads to a dangerous encounter with Draco Malfoy. They successfully pass Norbert to Charlie's friends, but they get caught by Mr Filch."
```

Comment:
Maybe you start this chapter with: 
During another visit at Hagrid's hut he reveals them... and ask them to help...

Also should mention that Ron is not capable of helping them as he got bit by the dragon, which explains why only Harry and Hermione get punished in the next chapter.